core/domain/entities/routing/data_nodes/core/CurrentUser.php

```diff
@@ -55,6 +61,7 @@ public function initialize()
         $this->addData('nickname', $current_user->nickname);
         $this->addData('username', $current_user->user_login);
         $this->addData('__typename', 'User');
+        $this->addDataNode($this->capabilities);
```

Comment:
As I said [here](https://github.com/eventespresso/barista/pull/314#discussion_r498063381), WP GQL already has that field, we should not override it.